components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/pom.xml

```diff
@@ -140,7 +140,6 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
Since the parent pom of the repo has the plugin enabled, let's remove the config from here.